pkg/cloudprovider/provider/kubevirt/provider.go

```diff
@@ -686,9 +701,14 @@ func (p *provider) newVirtualMachine(c *Config, pc *providerconfigtypes.Config,
 		annotations["kubevirt.io/ignitiondata"] = userdata
 	}
 
-	annotations["ovn.kubernetes.io/allow_live_migration"] = "true"
 	annotations["kubevirt.io/allow-pod-bridge-network-live-migration"] = "true"
 
+	if c.ProviderNetworkName == "KubeOVN" {
```

Comment:
Wouldn't it be better to compare the strings in lowercase?

Reply:
Well, this is not really a string comparison per se, it is semantically types comparison. However I see your point, I can lowercase it unconditionally